documentation/asciidoc/topics/con_operator_deployments.adoc

```diff
@@ -29,3 +31,12 @@ endif::community[]
 ifdef::downstream[]
 image::operator-managed-clusters.png[This illustration depicts how {ispn_operator} manages multiple clusters on {openshiftshort}.]
 endif::downstream[]
+
+= Resource reconciliation
+
+{ispn_operator} reconciles `Infinispan` custom resources such as the `Cache` CR with resources on your {brandname} cluster.
+
+Bidirectional reconciliation synchronizes your CRs with changes that you make to `Infinispan` resources through the {brandname} Console, command line interface (CLI), or other client application and vice versa.
+For example if you create a cache through the {brandname} Console then {ispn_operator} adds a declarative Kubernetes representation.
```

Comment:
It's probably worth noting somewhere that Cache CRs created via the ConfigListener have their owner reference set to the Infinispan CR which they are associated with, meaning that when the Infinispan CR is deleted the cache CR(s) will be as well. Conversely, Cache CRs created by the user are not deleted with the Infinispan CR.

Reply:
I've added that as a paragraph to this section. Maybe it's a bit wordy though.